### ISSUE TITLE
Question: Can I edit boot order ?

### ISSUE THREAD

**Issue by falconws** [kimchi-project/kimchi#1148]
Can I edit virtual machine boot order(priority) from kimchi Web UI ?
like this
https://stackoverflow.com/questions/19011159/how-to-set-boot-order-on-kvm-libvirt-virsh

**Comment by alinefm** [comment]
Hi @falconws 

That feature is not available on Kimchi UI yet. The backend API is already done but we are lacking the Ui code. Keep posted on next releases announcements to get it once it is done.

**Comment by alinefm** [comment]
![fb7fee5fb9c47b3](https://user-images.githubusercontent.com/4921718/27889767-d180fcc4-61c4-11e7-948e-384c32d3d63c.jpg)

There we go! :-)

Edit the guest you want to change the boot order and in General tab you can reorder the list by grabbing the lines and (un)select which you want to use.
